src/controllers/authenticationController.js

```diff
@@ -123,6 +124,7 @@ export default class AuthenticationController {
    * @param res
    */
   metadata(req: express$Request, res: express$Response) {
+    // $FlowFixMe
```

Comment:
what's the problem here?

Reply:
Actually our spid-passport package isn't covered by a flow-typed definition, we will have to create a new one.